lib/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,36 @@
+import { PropsWithChildren } from 'react'
+import './styles.css' // Importing the dedicated CSS file
+
+export interface TimelineEventProps {
+  date: string
+  title: string
+  description: string
```

Comment:
These would move to the subcomponent. Instead of date, could you call this prop aside. It won't always be a date, it can be any text.

Reply:
Done:
  aside: string
  title: string
  description: string